loader/status.go

```diff
@@ -66,6 +66,20 @@ func (l *Loader) PrintStatus(ctx context.Context) {
 		finishedSize := l.finishedDataSize.Load()
 		totalSize := l.totalDataSize.Load()
 		totalFileCount := l.totalFileCount.Load()
+
+		for db, tables := range l.dbTableDataFinishedSize {
+			for table, size := range tables {
+				curFinished := size.Load()
+				speed := float64(curFinished-l.dbTableDataLastFinishedSize[db][table]) / printStatusInterval.Seconds()
+				l.dbTableDataLastFinishedSize[db][table] = curFinished
+				var remainingSeconds float64
+				if speed > 0 {
```

Comment:
for a slow or not started table, speed is 0 and this `if` is skipped, so `remainingSeconds` is 0

Reply:
the lesson is, we should always think of the effect of`else` branch even there's no `else` branch

Reply:
this was very helpful to me